src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        return Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
Overloaded expression, please introduce some variables. Also Random should be class level variable